phpcs.xml.dist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
```

Comment:
You should add `phpcs.xml` to .gitignore so developers can override it locally without accidentally committing it.

Reply:
see #6